local_planner/src/nodes/common.h

```diff
@@ -10,58 +10,53 @@
 
 namespace avoidance {
 
-float distance2DPolar(int e1, int z1, int e2, int z2);
+struct PolarPoint {
+  float e;
+  float z;
+  float r;
+};
+
+float distance2DPolar(const PolarPoint& p1, const PolarPoint& p2);
 
 /**
 * @brief     Convertes the point from polar CS to cartesian CS
-* @param[in] e elevation angle (-90, 90]
-* @param[in] z azimuth angle (-180,180]
-* @param[in] radius
+* @param[in] p_pol polar point
```

Comment:
These comments are not so correct anymore. When I see a function `polarToCartesian()`, I expect it to take in a polar point, convert it to cartesian and return that. Why are there two arguments? This should be answered by the comments and preferably obvious from the function name. What the function actually does is add a polar unit vector to a cartesian point, returning the resulting 3D point in cartesian format. (Right? I may even be wrong here!)